dev-tools.asciidoc

```diff
@@ -358,10 +358,53 @@ $ npm -g install embark
+
+It is community-driven, lead by the https://zeppelin.solutions/[Zeppelin] developers, with hundreds of contributors from many Ethereum projects. The main focus of the framework is security, achieved by applying industry standard contract security patterns and best practices, taking advantage of all the experience the Zeppelin devs have gained from https://blog.zeppelin.solutions/tagged/security[auditing] a huge amount of contracts, and through the constant testing and auditing from the community that uses the framework as a base for their real world applications.
+
+The OpenZeppelin framework is the most widely used solution for Ethereum smart contracts. This is because the community discusses every proposed solution and learns from the implementation and integration of these solutions, which turns into a growing feedback loop that improves the existing contracts and finds new possibilities to combine them in a clear and secure architecture. It has resulted in the constant addition of new reusable contracts to solve bigger challenges or explore exciting new possibilities on top of the Ethereum blockchain. As mentioned before, the framework currently has an ample library of contracts ranging from implementations of `ERC20` and `ERC721` tokens, to many flavors of crowdsale models, to simple behaviors commonly found in contracts such as `Ownable`, `Pausable` or `LimitBalance`. The contracts in this repository in some cases even function as _de facto_ standard implementations.
+
+The framework has an MIT license, and all the contracts have been designed with a modular approach, to be easy to reuse and extend. These are clean and basic building blocks, ready to be used on your next Ethereum project. Let's set up the framework and build a simple crowdsale using the OpenZeppelin contracts, as an example to show you how easy it is, to stress on the importance of reusing secure components instead of writing them by yourself, and to give a small sample of the ideas that the Ethereum platform and its community are making possible.
+
+First, we will need to install the openzepelin-solidity library from npm into our workspace. The latest release as of the time of this writing is `v1.9.0`, so we will use that one.
+
+----
+mkdir sample-crowdsale
+cd sample-crowdsale
+npm install openzeppelin-solidity@1.9.0
+mkdir contracts
+----
+
+For the crowdsale, we will need to define a token that we will give to the investors in exchange for their ether. At the time of writing, OpenZeppelin includes multiple basic token contracts that follow the ERC20, ERC721 and ERC827 standards, with different characteristics for emission, limits, vesting, lifecycle, etc.
+
+Let's make an ERC20 token that's mintable, meaning that the initial supply starts at 0 and new tokens can be created by the token owner (in our case, the crowdsale contract) and assigned to the investors. For this, write the following into `contracts/SampleToken.sol`:
```

Comment:
For this ... => In order to do this, create a `contracts/SampleToken.sol` file with the following contents: